sycl/test/basic_tests/info.cpp

```diff
@@ -11,6 +11,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+#include "../helpers.hpp"
```

Comment:
Consider adding dir to the including directories in lit.cfg.py.

Reply:
@bjoernknafla Will you do it or we merging as is?

Reply:
I am currently experiencing many LIT failures on the `sycl` branch and am investigating before I can rebase.

As @bader pointed out in another PR, also there is no need to replace `assert` as it isn't turned off when not explicitly defining the `NDEBUG` preprocessor symbol. I was confused as CMake has extra behavior but these LIT tests do not go through CMake. I am considering rejecting and closing this PR.

Reply:
Sounds good to me.